app/app.iml

```diff
@@ -90,6 +90,7 @@
     <orderEntry type="library" exported="" name="parceler-api-0.2.16-SNAPSHOT" level="project" />
     <orderEntry type="library" exported="" name="support-annotations-21.0.3" level="project" />
     <orderEntry type="library" exported="" name="activeandroid-3.1.0-SNAPSHOT" level="project" />
+    <orderEntry type="library" exported="" name="picasso-2.5.0" level="project" />
```

Comment:
gives a simple api to background loading of images + handles re-use in list views (see `EventsAdapter` for usage).